### PR TITLE
added support for installation of metering via OperatorHub UI

### PR DESCRIPTION
**Pull request by pruan-rht**

### CODE REVIEW

**Comment by pruan-rht** [comment]
@yapei @akostadinov @liangxia PTAL.  Thanks!

File: lib/rules/web/admin_console/4.5/catalog.xyaml [hunk 1]
```diff
@@ -129,13 +129,17 @@ click_subscribe_button:
...
+    button_text: Install
+  action: click_button_text
 subscribe_using_default_params:
   action: wait_box_loaded
   action: click_subscribe_button
 set_custom_channel_and_subscribe:
   action: wait_box_loaded
   action: choose_update_channel
-  action: click_subscribe_button
+  action: click_install_button
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/catalog.xyaml:142]: in 4.5 `click_subscribe_button` just click on a button with text `Install`

```
click_subscribe_button:
  params:
    button_text: Install
  action: click_button_text
```

Comment (pruan-rht) [reply lib/rules/web/admin_console/4.5/catalog.xyaml:142]: @yapei hmm...in my branch that's not defined.  I guess it must've been udpated before I started this.

Comment (pruan-rht) [reply lib/rules/web/admin_console/4.5/catalog.xyaml:142]: @yapei, yeah that's the case, my orignal branch didn't have that.  Thanks for pointing it out, I'll revet that change
```
click_subscribe_button:
  params:
    button_text: Subscribe
  action: click_button_text
```

File: features/step_definitions/metering.rb [hunk 2]
```diff
@@ -1,23 +1,27 @@
 # helper step for metering scenarios
 require 'ssl'
+
 # For metering automation, we just install once and re-use the installation
 # unless the scenarios are for testing installation
 # checks that metering service is already installed
-Given /^metering service has been installed successfully(?: using (ansible|shell script|OLM))?$/ do |method|
+# shell => ./bin/hack.sh
```
Comment (liangxia) [inline features/step_definitions/metering.rb:7]: Is it still valid ?

Comment (liangxia) [reply features/step_definitions/metering.rb:7]: Then perhaps we need to update line 10, or else we are not able to use it.

Comment (pruan-rht) [reply features/step_definitions/metering.rb:7]: @liangxia we plan to phase that `hack.sh` script out.  That's why I took it out as an option.

Comment (liangxia) [reply features/step_definitions/metering.rb:7]: Or we should remove this line as well ?  It is confusing that we leave it there but removed it from line 10

Comment (pruan-rht) [reply features/step_definitions/metering.rb]: line removed... PTAL again. thx!

File: features/step_definitions/metering.rb [hunk 3]
```diff
@@ -27,19 +30,17 @@
...
-    cb[:metering_namespace] = project(namespace)
     metering_name = metering_config.raw_resource['metadata']['name']
   end
 
   # change project context
-  unless cb.metering_namespace
+  unless metering_config(namespace).exists?
     case method
     when "shell script"
       step %Q/metering service is installed using shell script/
```
Comment (liangxia) [inline features/step_definitions/metering.rb:40]: Those lines should be removed as well.

Comment (pruan-rht) [reply features/step_definitions/metering.rb]: ok, updated.

**Pull request merged**